internal/server/models/access_key.go

```diff
@@ -13,8 +13,9 @@ var (
 )
 
 const (
-	ScopePasswordReset        string = "password-reset"
-	ScopeAllowCreateAccessKey string = "create-key"
+	ScopePasswordReset                 string = "password-reset"
+	ScopeAllowCreateAccessKey          string = "create-key"
+	ScopeAllowApproveDeviceFlowRequest string = "approve-device-flow-request"
```

Comment:
These don't seem like they are really different scopes. Are we assigning these two scopes in all the same cases?

Device flow is effectively a way of getting an access key in the CLI. Can we continue to use `ScopeAllowCreateAccessKey` for approving of the request?

Reply:
They are close, but access keys resulting from logging with device flow should be able to:
1. Create an access key (since the user ran `infra login`), they should be able to run `infra keys add`
2. But not approve new device flows, since this would mean the user could continue to approve their own device flow requests

Reply:
> But not approve new device flows, since this would mean the user could continue to approve their own device flow requests

Why is that a problem?